client/block-builder/src/lib.rs

```diff
@@ -198,6 +198,13 @@ where
 		})
 	}
 
+	/// Called after inherents but before extrinsics have been applied.
+	pub fn after_inherents(&self) -> Result<BlockAfterInherentsMode, Error> {
+		self.api
+			.after_inherents_with_context(self.parent_hash, ExecutionContext::BlockConstruction)
+			.map_err(Into::into)
```

Comment:
Should check the version of the runtime api and not return an error if the version introducing `after_inherents` is not yet available.

Reply:
Changed